opensearch-project/i18n-plugin
# Bump to 3.0.0 and add `es` translation

### Description
Bumped with:
`yarn plugin-helpers version --sync legacy`

Translation file generated from root level OSD core with: `./scripts/use_node scripts/i18n_extract.js --output-dir plugins/dashboards-i18n/translations/`

To test:
* Clone this repo into `OSD core/plugins`
* `yarn osd bootstrap`
* Start OpenSearch with `yarn opensearch snapshot`
* Start OpenSearch Dashboards with `yarn start --i18n.locale=es`

Appended to some string with `x` to verify working functionality.

### Issues Resolved
n/a

### Check List
- [x] Existing locale: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/packages/osd-i18n/src/core/locales.js#L27

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
